components/pvcviewer-controller/config/manager/kustomization.yaml

```diff
@@ -1,2 +1,7 @@
 resources:
 - manager.yaml
+
+configMapGenerator:
+- name: manager-config
+  envs:
+  - params.env
```

Comment:
In order for the environment variables from this config map to be used, we 'll need to update also the deployment of the controller https://github.com/madmecodes/kubeflow/blob/df73d2217f011642003520c3d16a371beadf8f9e/components/pvcviewer-controller/config/manager/manager.yaml#L15-L16. Otherwise, I don't think this will have any effect.

Reply:
Applies to tensorboard-controller as well.

Reply:
I think this is still not resolved for the same reasons as https://github.com/kubeflow/kubeflow/pull/7736/files#r2266986072. Those envvars will not be propagated to the deployment IIUC. Have you tried deploying manifests from this repo to confirm?